### PR TITLE
feat(203): adding pisp demonstration

### PR DESCRIPTION
**Pull request by pawarspeaks**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "ml-testing-toolkit-ui",
-  "version": "15.5.3",
+  "version": "15.5.0",
```
Comment (elnyry-sam-k) [inline package.json:3]: please revert

File: package.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
   "description": "Mojaloop Testing Toolkit Web User Interface",
   "main": "index.js",
+  "proxy": "http://localhost:4040",
```
Comment (elnyry-sam-k) [inline package.json:6]: Is this required?

File: src/index.js [hunk 3]
```diff
@@ -25,6 +25,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import React, { useState } from 'react';
 import ReactDOM from 'react-dom';
 import reportWebVitals from './reportWebVitals';
+// import { format } from 'util';
```
Comment (elnyry-sam-k) [inline src/index.js:28]: please delete

File: src/views/outbound/TestCaseDemoViewer.jsx [hunk 4]
```diff
@@ -22,6 +22,7 @@
  --------------
  ******/
 import React from 'react';
+// import _ from 'lodash';
```
Comment (elnyry-sam-k) [inline src/views/outbound/TestCaseDemoViewer.jsx:25]: please delete

**Review by elnyry-sam-k — CHANGES_REQUESTED** [review]
Some minor comments

**Comment by elnyry-sam-k** [comment]
Also, as a general comment - we likely don't need the settlement related files here, isn't it?

File: package.json (hunk 1 above)
Comment (pawarspeaks) [reply package.json:3]: done!

File: package.json (hunk 2 above)
Comment (pawarspeaks) [reply package.json:6]: yes actually for hitting api endpoints live at pisp-ttk-sandbox docker container, either way it was giving cors errors

**Review by elnyry-sam-k — APPROVED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**